charter.html

```diff
@@ -0,0 +1,330 @@
+      that assist in our mission. The focus is to enable scaled anti-fraud and
+      solutions that are applicable across the ecosystem. 
+    </p>
+    <h3 id="out-of-scope">
+      Out of Scope
+    </h3>
+    <p>
+      Carve-outs (exceptions to restrictions/limitations) for other web
+      platform work should be brought up in the appropriate group, rather than
+      as part of the Anti-Fraud CG. Features which don’t interact with the
+      browser (or similar user agent) should be proposed in other standards
+      bodies (IETF, etc).
+    </p>
+    <h2 id="meetings">
+      Meetings
+    </h2>
+    <p>
+      Meetings will be held biweekly on {TODO: INSERT DATE}, lasting for one
+      hour. Meetings will be run by one of the Chairs and will request scribes
+      to take meeting notes, which will be posted in the
+      <a href="https://github.com/antifraudcg/meetings">Meetings</a> repository
+       after the meeting. Any action items from the meeting will also be posted
+       to the Community Group mailing list to bring them to the attention of
+       people who did not attend.
+    </p>
+    <p>
+      Infrequently, face-to-face meetings will be held for the Anti-Fraud
+      Community Group, allowing for more collaboration and iteration between
+      Community Group members.
+    </p>
+    <h2 id="deliverables">
+      Deliverables
+    </h2>
+    <h3 id="work-items">
+      Current Work Items
+    </h3>
+    <p>
+      There are currently no work items that have been adopted by the Community
+      Group. See <a href="#proposals">Proposals</a> for the process of adopting
+      a proposal into the Community Group.
+    </p>
+    <p>
+      Once a work item is ready for standardization and there is
+      <a href="#decision">consensus</a> that the work item is ready to
+      be migrated to a standards body, the Chairs will collaborate with the
+      Community Group to prepare a Community Group Report on the item, and with
+      the Editors of the item to forward it and the Report to the appropriate
+      standards body.
+    </p>
+    <h3 id="proposals">
+      Proposals
+    </h3>
+    <p>
+      Anyone can submit proposals to the proposals repository. If there is
+      substantial documentation/work, the proposal authors can request the
+      Chairs set-up a dedicated repository for a proposal.
+    </p>
+    <p>
+      A proposal can be withdrawn by its authors, or closed if there's
+      consensus that the proposal is out of scope or the number of authors has
+      dropped to zero. The Chairs will check-in with active proposals to ensure
+      the set of active authors is kept up-to-date.
+    </p>
+    <p>
+      Generally proposals should include an <i>Explainer</i> document describing
+      the changes to the web platform and impact of the proposal.
+    </p>
+    <p>
+      Work items are proposals that can lead to interoperable standards/APIs.
```

Comment:
Early sections of the charter suggest that some deliverables of the group will be documentation of scenarios and use cases, which wouldn't be direct feature/API proposals.

Reply:
I wonder if we should have work items encompass both types of deliverables, and then the last stage (standardization for standards/APIs) doesn't apply or some other path is described for non-API "work items".